### ISSUE TITLE
Searching track names not supported

### ISSUE THREAD

**Issue by astrowave** [forslund/mopidy_skill#22]
Most of the time Mycroft won't play songs searched by name because this skill doesn't support searching by track.

Mopidy does support searching by track names but I haven't been able to get it working yet.

```
    d['method'] = 'core.library.search'
    d['params'] = {'track_name': search_for_track}
    r = requests.post(url, headers={"content-type":"application/json"}, data=json.dumps(d))
    return r.json()
```

**Comment by forslund** [comment]
Hi thanks for reporting, playing single tracks was never meant to be a feature of this skill due to how it handles caching the information.

I started looking at moving to the Common Play Framework (but never finished) and in that context it makes more sense allowing search for specific tracks.

I'll try finish the move to CPS this weekend and see if track searching can be added as well.

**Comment by astrowave** [comment]
You're welcome.

I've actually been working on single track this lunchtime and it's almost complete save for a small issue.  I'll update my fork when I get in from work and you can take a look.

**Comment by forslund** [comment]
Ah Excellent, thank you very much :)

**Comment by astrowave** [comment]
I've submitted a pull request for this but  it could do with testing for spotify and gmusic compatibility as I don't have those accounts to test against.  It shouldn't impact that side of it but best to be thorough.

**Comment by forslund** [comment]
Resolved by #23
